actions/filter-files/filter-files.test.mjs

```diff
@@ -13,7 +13,6 @@ describe("filterFiles", () => {
         const result = filterFiles({
             globsRaw: `packages/*\n!packages/*.json`,
             inputFiles,
-            matchAllGlobs: true,
```

Comment:
before I had to set `matchAllGlobs` to true in order for the "NOT glob" to be usefully applied.